inductiva/_cli/cmd_resources/terminate.py

```diff
@@ -1,54 +1,78 @@
 """CLI commands to terminate computational resources."""
+import sys
 
 from inductiva import resources
+from inductiva.utils import input_functions
+from ...localization import translator as __
 
 
 def terminate_machine_group(args):
     """Terminate one or all computational resoruces."""
+    names = args.name
+    all_names = args.all
+    confirm = args.confirm
+
+    if names and all_names:
+        print(
+            "inductiva resources terminate: error: "
+            "argument name not allowed with argument --all",
+            file=sys.stderr)
+        return 1
+
     active_machines = resources.machine_groups.get()
-    counter = 0
 
     if not active_machines:
-        return
-
-    # Doesn't run in case --all is not passed.
-    # Confirm the termination of all machines.
-    if args.terminate_all:
-        prompt = input("Confirm the termination of all active machines? (y/n)")
-        confirm = prompt.lower() in ["y", "ye", "yes"]
-        if confirm:
-            print("Terminating all active computational resources.")
-        else:
-            print("Aborting the termination of resources.")
-            return
-
-    for machine in active_machines:
-        if args.terminate_all:
-            machine.terminate()
-        elif machine.name == args.name:
-            machine.terminate()
-            break
-        else:
-            counter += 1
-
-    if counter == len(active_machines):
-        print("No active computational resources found with the "
-              f"name {args.name}.")
+        return 0
+
+    # dict to map from name to machine
+    name_to_machine = {machine.name: machine for machine in active_machines}
+    active_machine_names = name_to_machine.keys()
+    target_machine_names = set(
+        names)  # the user can give the same name multiple times!!
+    invalid_names = target_machine_names.difference(active_machine_names)
+
+    if invalid_names:
+        for name in invalid_names:
+            print(f"Resource {name} does not exist.")
+        print("Aborting.")
+        return 1
```

Comment:
TBH, I'm not sure what should be the default behaviour, but it seems weird that we would abort everything because one name failed.

Reply:
Moreover, thinking about this a bit more, a `invalid_name` may come from an error on the name, or the machines being already terminated. At the moment, we cannot know if it has been terminated, because that is not saved in the DB, but I see a possibility where the same configuration (including the name) are re-usable by the user, so the information of it being already terminated is important if they want to raise again.

Just a bit of rambling... WDYT?

Reply:
Well, I'm a fan of terminating everything if one fails. As a user, I would want my command to be fully complete or none at all. I don't want my environment in a gray area where I don't really know what was done or wasn't (I mean, I can know if I read the full logs, but that can be massive).

Reply:
That is fine by me, but then we should not do a loop to add more logs to the table... It should just be a single message on top and then list the invalid names below.